hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/OzoneManagerStateMachine.java

```diff
@@ -103,6 +111,16 @@ public void initialize(RaftServer server, RaftGroupId id,
     });
   }
 
+  @Override
+  public SnapshotInfo getLatestSnapshot() {
+    return snapshotInfo;
+  }
+
+  @Override
+  public void notifyIndexUpdate(long term, long index) {
```

Comment:
I see we ignore the supplied index. I guess we don't need to update it - why is that?

Reply:
The supplied index here is the latest log index. We do not want the snapshotIndex to be updated to the last log index. It should be the last applied index which is persisted to the DB. So we don't update that here.